src/pages/messages/TasksPage.tsx

```diff
@@ -172,10 +169,10 @@ const TasksPage = () => {
                     setTicketTypes({ ...ticketTypes, publishingRequest: !ticketTypes.publishingRequest });
                     resetPaginationAndNavigate(searchParams, navigate);
                   }}>
-                  <InsertDriveFileOutlinedIcon fontSize="small" />
-                  {ticketTypes.publishingRequest && publishingRequestCount
-                    ? `${t('my_page.messages.types.PublishingRequest')} (${publishingRequestCount})`
-                    : t('my_page.messages.types.PublishingRequest')}
+                  <TicketTypeTag
+                    count={ticketTypes.publishingRequest && publishingRequestCount ? publishingRequestCount : undefined}
+                    type={TicketTypeEnum.PublishingRequest}
+                  />
```

Comment:
Using new component for this text and icon with colored background:

<img width="305" height="223" alt="image" src="https://github.com/user-attachments/assets/f4afe7be-adde-40c5-99af-21af24a0fce6" />

So that it can be used in other contexts (without count and without text)